### PR TITLE
net-dns/{bind,bind-tools} 9.18.0 bump

### PR DESCRIPTION
**Pull request by erayaslan**
see https://bugs.gentoo.org/832218

### CODE REVIEW

**Comment by mikhailkoliada** [comment]
@erayaslan could you provide a diff?

**Comment by erayaslan** [comment]
[bind-9.18.0.diff.txt](https://github.com/gentoo/gentoo/files/7963406/bind-9.18.0.diff.txt)

[bind-tools-9.18.0.diff.txt](https://github.com/gentoo/gentoo/files/7963407/bind-tools-9.18.0.diff.txt)

diffs attached

**Comment by mikhailkoliada** [comment]
@erayaslan Thanks!

I would not like to drop chroot as this is widely-used, rather add it with a systemd unit too

**Comment by mikhailkoliada** [comment]
@erayaslan I would also probably drop static-libs entirely, I can not think of any use cases for this in bind

File: net-dns/bind/bind-9.18.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+	dnstap? ( dev-libs/fstrm dev-libs/protobuf-c )
+	doh? ( net-libs/nghttp2 )
+	geoip? ( dev-libs/libmaxminddb )
+	gssapi? ( virtual/krb5 )
+	idn? ( net-dns/libidn2 )
+	lmdb? ( dev-db/lmdb )
+	xml? ( dev-libs/libxml2 )
+"
+BDEPEND="
+	test-extra? (
```
Comment (mikhailkoliada) [inline net-dns/bind/bind-9.18.0.ebuild:42]: I'd leave it under the test USE flag, to generalize the installations and not to make additional checks what to enable/disable

**Review by mikhailkoliada — CHANGES_REQUESTED** [review]

File: net-dns/bind/bind-9.18.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+	selinux? ( sec-policy/selinux-bind )
+	sys-process/psmisc
+	!net-dns/bind-tools
+"
+
+S="${WORKDIR}/${MY_P}"
+
+src_configure() {
+	local myeconfargs=(
+		--prefix="${EPREFIX}"/usr
```
Comment (mikhailkoliada) [inline net-dns/bind/bind-9.18.0.ebuild:59]: Eprefix is needed for /etc and /var dirs too I believe

**Review by mikhailkoliada — CHANGES_REQUESTED** [review]

File: net-dns/bind-tools/bind-tools-9.18.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+	for tool in cds dsfromkey importkey keyfromlabel keygen \
+		revoke settime signzone verify; do
+		doman ${man_dir}/dnssec-"${tool}".1
+	done
+
+	emake DESTDIR="${D}" -C bin/tools/ install
+	doman ${man_dir}/{arpaname,mdig,named-journalprint,named-rrchecker,nsec3hash}.1
+
+	# just leave the tools to be installed
+	rm -rf "${D}"/usr/include/
```
Comment (mikhailkoliada) [inline net-dns/bind-tools/bind-tools-9.18.0.ebuild:114]: `|| die`

**Review by mikhailkoliada — CHANGES_REQUESTED** [review]

**Comment by erayaslan** [comment]
> I would not like to drop chroot as this is widely-used, rather add it with a systemd unit too

sure, your call. but I'm not going to do it as I do not think it is worth the code complexity and maintenance burden.

hopefully, the above notes will help with the bump.

closing

File: net-dns/bind-tools/bind-tools-9.18.0.ebuild (hunk 3 above)
Comment (erayaslan) [reply net-dns/bind-tools/bind-tools-9.18.0.ebuild:114]: with -f I do not think it will ever fail

File: net-dns/bind/bind-9.18.0.ebuild (hunk 1 above)
Comment (erayaslan) [reply net-dns/bind/bind-9.18.0.ebuild:42]: the reason was you can not run

`# USE=test ebuild bind-9.18.0.ebuild test`

as that results in circular dependencies with pytest.  so this was a dirty hack but yep maybe not the best idea

**Pull request closed without merging**